### PR TITLE
Feature/at16 352 metadatos

### PR DESCRIPTION
**Pull request by IvanLazo24**
review the new changes of metadatos

### CODE REVIEW

File: METADATOS/Meta 6.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+import subprocess
+import os
+
+##CREAMOS NUESTRA CLASE
+class Metadata:
+
+### DEFINIMOS LAS ENTRADAS DE LA CLASE
+    def __init__(self,meti):
+        self.path = "C:/IVAN/CODING/Metadatos/filein"
```
Comment (FreddyClaros) [inline METADATOS/Meta 6.py:22]: Can you change this paths lo local paths instead of absolute paths please

File: METADATOS/Meta 6.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                    process = subprocess.run([exe, str(self.path) + "/" + str(file), '-o',
+                                               str(self.direct) + namef[0] + '.' + str(self.meti)])
+                    ## imprimimos el estado de cada conversion
+                    print(process.stdout)
+
+
+######### LLAMAMOS A NUESTRA CLASE CON LOS VALORES DE LA DIRECCION DE ENTRADA, LA EXTENCION Y LA
+# SALIDA
+prueba = Metadata('xmp')
+prueba.select_file()
```
Comment (FreddyClaros) [inline METADATOS/Meta 6.py:60]: add an extra line at the end please

File: METADATOS/Meta 6.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+#                namef = name[:-5]## eliminamos la extencion del archivo en el caso del ejemplo el
+                # ".jpg"
+                if self.meti == 'json':
+                ##procesamos cada archivo con el Exiftool.exe y sacamos sus dependencias en
+                # formato JSON creando un archivo.JSON para cada imagen
+                    process = subprocess.run([exe,str(self.path) + "/" + str(file), '-' + str(
+                        self.meti), '-W+!', str(self.direct) + namef[0] + '.' + str(self.meti)])
+                ## imprimimos el estado de cada conversion
+                    print(process.stdout)
+                ###--******************************************************************************
```
Comment (FreddyClaros) [inline METADATOS/Meta 6.py:43]: what's the purpose of this lines?

**Review by Amilcar-B — COMMENTED** [review]
it must have a space below finish coding

File: METADATOS/metaout/leer2.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+Esta es la carpeta en donde estan las salidas de los archivos JSON
```
Comment (FreddyClaros) [inline METADATOS/metaout/leer2.txt:1]: not sure if "Metaout" folder it's a good name, maybe change it for something like metadata_output or something like that

File: METADATOS/imagenes/Leer.txt [hunk 5]
```diff
@@ -0,0 +1 @@
+esta catpeta es para guardar los archivos a revisar
```
Comment (FreddyClaros) [inline METADATOS/imagenes/Leer.txt:1]: The name of some folders are in Spanish, please change it to English

File: METADATOS/Meta 6.py (hunk 1 above)
Comment (abrahamUrquiola) [reply METADATOS/Meta 6.py:22]: is it okay that the comments are in spanish? review please

**Pull request closed without merging**